src/test/java/DeadlineTest.java

```diff
@@ -0,0 +1,21 @@
+import org.junit.jupiter.api.Test;
+import duke.task.Deadline;
+
+import java.time.LocalDate;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DeadlineTest {
+    @Test
+    public void createNewDeadlineTest(){
```

Comment:
Perhaps you can consider naming your tests in the following format `featureUnderTest_testScenario_expectedBehavior()`? I noticed the same issue in several other places too.